amy/extforms/forms.py

```diff
@@ -88,6 +90,7 @@ class Meta:
         }
 
     def __init__(self, *args, **kwargs):
+        self.request_http = kwargs.pop("request", None)
```

Comment:
This name is a little bit off. Usually it was named `self.request`.

Reply:
I did this to avoid ambiguity between the http request and the training request that this form describes. I'm not entirely happy with the break from convention but I think it might prevent future confusion